Makefile

```diff
@@ -77,7 +80,7 @@ a11y: test-build pally-conf
 
 # Note: `run` executes `node demo/app`, which fires up express, then deploys
 # a test static site to s3, then exits, freeing the process to execute `nightwatch a11y`.
-test: developer-note verify pally-conf test-server test-browser test-build test-webpack run nightwatch a11y
+test: developer-note verify pally-conf test-server test-browser test-build test-webpack run nightwatch a11y build-webpack
```

Comment:
Actually, I think I'd rather it just be part of the deploy step. I'd be nervous about doing the `build-production` and then `test` incase `test` generates stuff we don't want to be deployed.